horace_core/sqw/@DnDBase/DnDBase.m

```diff
@@ -2,7 +2,7 @@
     % DnDBase Abstract base class for n-dimensional DnD object
 
     properties(Access = protected)
-        data % dnd_sqw_data instance
+        % removed, replaced with data_ on the superclass data % dnd_sqw_data instance
```

Comment:
is this comment helpful? The whole `properties(protected)` block can be deleted.

Reply:
@nickbattam-tessella It will cease to be helpful when all the sqw conversion has taken place. At the moment it provides a degree of traceability during the time I am trying to make this work